service/collector/chart_config_resource.go

```diff
@@ -27,8 +27,8 @@ var (
 		nil,
 	)
 
-	cordonExpireTimeDesc = prometheus.NewDesc(
-		prometheus.BuildFQName(Namespace, "", "cordon_expire_time_seconds"),
+	chartConfigCordonExpireTimeDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(Namespace, "", "chartconfig_cordon_expire_time_seconds"),
```

Comment:
After this, I need to update g8s-prometheus update.

Reply:
I think subsystem should be `chart_config` and the metric name `cordon_expire_time_seconds`.

Reply:
Thanks for the idea, I didn't realize we could utilize the subdomain in here.

Reply:
Also, we can use this subsystem, for `chartconfig_status` metric.

Reply:
@xh3b4sd I remove the underline character inside `chart_config` you suggest since we have alerting rules that using metrics. is that okay for you?

Reply:
It is not perfect but it works. Thanks for taking care mate!